src/components/structures/auth/Registration.js

```diff
@@ -549,6 +560,15 @@ module.exports = React.createClass({
             { _t('Sign in instead') }
         </a>;
 
+        let goBack = <a className="mx_AuthBody_changeFlow" onClick={this.onGoToFormClicked} href="#">
```

Comment:
nit: only assign goBack if it'll be used? `let foo; if (bar) foo = <div/>` is also the pattern we use elsewhere.

Reply:
it seemed a bit weird to have it default off when the majority of cases use it, but I also agree that going against the grain here is more confusing.